dhall-json/src/Dhall/JSONToDhall.hs

```diff
@@ -191,6 +191,21 @@
 > $ json-to-dhall --no-keyval-maps 'List { mapKey : Text, mapValue : Text }' <<< '{ "foo": "bar" }'
 > Error: Homogeneous JSON map objects cannot be converted to Dhall association lists under --no-keyval-arrays flag
 
+    If you have an `Record` schema with a property of type `List` and omit that property in the JSON,
+    you'll get an error:
```

Comment:
"property" is a term from the JSON world, right? I believe the corresponding Dhall term is "record field".

```suggestion
    If your schema is a record with a `List` field and omit that field in the JSON,
    you'll get an error:
```